redbeams/src/main/java/com/sequenceiq/redbeams/domain/DatabaseServerConfig.java

```diff
@@ -228,4 +231,29 @@ public void setEnvironmentId(String environmentId) {
         this.environmentId = environmentId;
     }
 
+    /**
+     * Creates a database with a database name and type from the database server config.
+     *
+     * @param databaseName the name of the database
+     * @param type         the type of database
+     * @return a DatabaseConfig
+     */
+    public DatabaseConfig createDatabaseConfig(String databaseName, String type) {
+        DatabaseConfig databaseConfig = new DatabaseConfig();
+
+        databaseConfig.setDatabaseVendor(databaseVendor);
+        databaseConfig.setName(databaseName);
+        databaseConfig.setDescription(description);
+        databaseConfig.setConnectionURL(new DatabaseCommon().getJdbcConnectionUrl(databaseVendor.jdbcUrlDriverId(),
+            host, port, Optional.of(databaseName)));
+        databaseConfig.setConnectionDriver(connectionDriver);
+        databaseConfig.setConnectionUserName(connectionUserName.getRaw());
+        databaseConfig.setConnectionPassword(connectionPassword.getRaw());
+        databaseConfig.setStatus(ResourceStatus.USER_MANAGED);
```

Comment:
I'm wondering if this should be an argument to the method. Probably OK to leave it now. Just thinking about how this might be used later.

Reply:
Yeah, this isn't 100% great, but for the purposes of simplifying this for now I vote to leave and refactor as needed.